### ISSUE TITLE
hookTimeOut errors when defining two different mongo connections

### ISSUE THREAD

**Issue by alexklibisz** [balderdashy/sails-mongo#305]
When I configure two different mongo connections in config/connections.js:
```
mongoProd: {
    adapter: 'sails-mongo',
    host: '10.132.xxx.yyy',
    port: 27017,
    user: 'dbUser',
    password: 'dbPass',
    database: 'dbName'
  },
  mongoLocal: {
    adapter: 'sails-mongo',
    host: 'localhost',
    port: 27017,
    user: 'dbUser',
    password: 'dbPass',
    database: 'dbName' 
 }
```
And run my application (locally - should use the second connection):
```
> sails lift
```
I get one of the the following errors every time:

```
error: Error: The hook `orm` is taking too long to load.
```
```
error: Error: The hook `pubsub` is taking too long to load.
```

I've tried creating the files config/orm.js and config/pubsub.js to increase the (per http://stackoverflow.com/questions/28524926/the-hook-orm-taking-too-long-to-load), and added times up to 120000, but the problem still persists.

I've made sure that I'm configuring the correct connection string in the development.js and production.js files as well. And my models.js migrate option is set to "safe".

If I comment out either of the two connections, everything works again and doesn't require the hook timeout increases.

**Comment by sailsbot** [comment]
Thanks for posting, @alexklibisz.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
